src/chainstate/stacks/boot/mod.rs

```diff
@@ -813,6 +835,34 @@ pub mod test {
         )
     }
 
+    pub fn make_pox_2_lockup(
+        key: &StacksPrivateKey,
+        nonce: u64,
+        amount: u128,
+        addr_version: AddressHashMode,
+        addr_bytes: Hash160,
+        lock_period: u128,
+        burn_ht: u64,
+    ) -> StacksTransaction {
+        // (define-public (stack-stx (amount-ustx uint)
+        //                           (pox-addr (tuple (version (buff 1)) (hashbytes (buff 20))))
+        //                           (lock-period uint))
+        let payload = TransactionPayload::new_contract_call(
+            boot_code_test_addr(),
+            "pox-2",
```

Comment:
Minor style nit -- should probably have the contract name as a constant somewhere.

Reply:
Addressed in d6458eccc939ccf7806470012568b90a5470fda9